dash/examples/bench.10.summa/main.cpp

```diff
@@ -335,7 +335,7 @@ void perform_test(
 
   if (myid == 0) {
     if (iteration == 0) {
-      cout << "-- Pattern: " << pattern << endl
+      cout << "-- Pattern: " << dash::internal::typestr(pattern) << endl
```

Comment:
Oh yes, will add Doxygen comments for the new features. Note that namespace `internal` is excluded from API docs for obvious reasons. Perhaps some helpers like `typestr` should be public instead.